pytensor/scalar/math.py

```diff
@@ -684,6 +684,27 @@ def __hash__(self):
 gammaincc = GammaIncC(upgrade_to_float, name="gammaincc")
 
 
+class GammaIncInv(BinaryScalarOp):
```

Comment:
Wolfram suggests the derivative wrt to `s` (I don't know if that's k or x) is relatively simple, so it would be nice to add

See scope>differentiation 

https://reference.wolfram.com/language/ref/InverseGammaRegularized.html

Would also be nice to check if there are simple formula for the derivatives of the invbetainc

Reply:
Thanks for the pointer!! I feel bad for not checking it before, will definitely work on it

Reply:
No worries, that's how we all start